raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -1966,6 +1967,82 @@ def test_imbalance_penalty_at_insufficent_transfer_balance():
     assert not pair
 
 
+def test_imbalance_penalty_at_insufficent_mediator_balance():
+    """
+    Test that having an imbalance penalty fee during a transfer where mediator has
+    insufficient balance does not throw an UndefinedMediationFee exception from
+    the state machine.
+    """
+    payer_transfer = create(
+        LockedTransferSignedStateProperties(amount=10, initiator=HOP1, target=ADDR, expiration=50)
+    )
+
+    imbalance_penalty = calculate_imbalance_fees(
+        channel_capacity=20, proportional_imbalance_fee=4000
```

Comment:
Is that even a fee of one token wei? 20*4000/1e6=0,08

Reply:
For this test, it only matters that we try to calculate the IP and check is that succeeds. The result does not actually matter.

Reply:
Ok, can you add a comment about that?

Reply:
I've added such a comment to all three tests where this is the case. I'll try to clean up the tests later, so that there is less code and redundancy.

Reply:
Thanks!